releases.md

```diff
@@ -5,7 +5,7 @@ permalink: /releases/
 ---
 
 # Latest Stable
-The most recent stable release is {{ site.github.latest_release.tag_name }} {{ site.github.latest_release.name }}.  We recommend stable for the most bug-free experience possible.  You can download it from the links below.
+The most recent stable release is {{ site.github.latest_release.tag_name }} {{ site.github.latest_release.name }}.  We recommend stable for the most bug-free experience possible.  You can download it from the links below.  Note: downloads with 'Curses' in the name include a version that can be played in console and has no graphics. Downloads with 'Tiles' in the name includes a version of the game that has graphics.
```

Comment:
No need to use double spaces in md (html).
```suggestion
The most recent stable release is {{ site.github.latest_release.tag_name }} {{ site.github.latest_release.name }}. We recommend stable for the most bug-free experience possible. You can download it from the links below. Note: downloads with 'Curses' in the name include a version that can be played in console and has no graphics. Downloads with 'Tiles' in the name includes a version of the game that has graphics.
```

Reply:
Thanks